symfony/framework-bundle/4.2/src/Kernel.php

```diff
@@ -31,7 +29,7 @@ public function registerBundles()
     {
         $contents = require $this->getProjectDir().'/config/bundles.php';
         foreach ($contents as $class => $envs) {
-            if (($envs['all'] ?? false) && ($envs[$this->environment] ?? true) || ($envs[$this->environment] ?? false)) {
+            if (isset($envs['all']) || isset($envs[$this->environment])) {
```

Comment:
Why did you revert this change?

Reply:
to make the code the same in the v3.3 vs v4.2 recipes - and because the code with isset is more readable.

Reply:
But wasn't it fixing an actual bug? 

On my end, I don't even understand the version with `??`, too tricky for me, so big :+1: for readability.

Reply:
if it was a fix, it as not applied to v3.3 :)

Reply:
Of course, the issue was #383

Reply:
OK, fixed

Reply:
That's more readable than the previous solution :+1: Thx!

Reply:
Yep, new solution is much better 👍